docs/deploying-infra.md

```diff
@@ -62,10 +62,25 @@ az account set -s <subscription_id>
 
 For end-to-end deployment, you can either choose to run the `setup.sh` script that will take care of deploying all the services, building the docker images and configuring the variables or run the scripts individually as shown below.
 
+Note: For US Government Cloud, 
```

Comment:
Maybe refer to this as sovereign clouds, or custom cloud? There's also the German cloud, Mooncake, Azure Stack environments. It's possible in any of these that the features are not supported.

Reply:
Updated to Sovereign clouds.